### PR TITLE
Added publications and moved some content

### PR DESCRIPTION
**Pull request by JadeHotchkiss**
Added 3 publications.
Moved some info above the list of dependencies (not sure whether this will help them be visible instead of dropping off the page?)

### CODE REVIEW

File: ontology/scdo.md [hunk 1]
```diff
@@ -19,6 +19,19 @@ homepage: https://scdontology.h3abionet.org/
 products:
   - id: scdo.owl
   - id: scdo.obo
+publications: 
+   - id:  https://doi.org/10.1093/database/baac014
```
Comment (cthoyt) [inline ontology/scdo.md:23]:
```suggestion
  - id: https://www.ncbi.nlm.nih.gov/pubmed/35363306

File: ontology/scdo.md [hunk 2]
```diff
@@ -19,6 +19,19 @@ homepage: https://scdontology.h3abionet.org/
...
+     title: "The Sickle Cell Disease Ontology: recent development and expansion of the universal sickle cell knowledge representation."
+   - id: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7549008/
```
Comment (cthoyt) [inline ontology/scdo.md:25]:
```suggestion
  - id: https://www.ncbi.nlm.nih.gov/pubmed/33021900
```

File: ontology/scdo.md [hunk 3]
```diff
@@ -19,6 +19,19 @@ homepage: https://scdontology.h3abionet.org/
...
+     title: "The Sickle Cell Disease Ontology: Enabling Collaborative Research and Co-Designing of New Planetary Health Applications."
+   - id: https://academic.oup.com/database/article/doi/10.1093/database/baz118/5626537
```
Comment (cthoyt) [inline ontology/scdo.md:27]:
```suggestion
  - id: https://www.ncbi.nlm.nih.gov/pubmed/31769834

File: ontology/scdo.md [hunk 4]
```diff
@@ -19,6 +19,19 @@ homepage: https://scdontology.h3abionet.org/
...
+   - id:  https://doi.org/10.1093/database/baac014
+     title: "The Sickle Cell Disease Ontology: recent development and expansion of the universal sickle cell knowledge representation."
+   - id: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7549008/
+     title: "The Sickle Cell Disease Ontology: Enabling Collaborative Research and Co-Designing of New Planetary Health Applications."
+   - id: https://academic.oup.com/database/article/doi/10.1093/database/baz118/5626537 
+     title: "The Sickle Cell Disease Ontology: enabling universal sickle cell-based knowledge representation."
+tracker: https://github.com/scdodev/scdo-ontology/issues
+license:
+  url: https://www.gnu.org/licenses/gpl-3.0.en.html
+  label: GPL-3.0
```
Comment (cthoyt) [inline ontology/scdo.md:32]: any change you can change this to an appropriate license for data from Creative Commons like CC BY 4.0?

Comment (JadeHotchkiss) [reply ontology/scdo.md:32]: I will check and get back to you.

**Review by matentzn — APPROVED** [review]

**Pull request merged**